core.js

```diff
@@ -27,14 +28,6 @@ var methodMapping = {
 	}
 };
 
-
-// Get a global reference.
-var GLOBAL = typeof global !== "undefined"? global : window;
-
-// valid form types in addition to plain objects as URL search params
-// https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/send
-var formTypes = ['ArrayBuffer', 'ArrayBufferView', 'Blob', 'FormData'];
-
```

Comment:
This change basically switches from explicitly checking for different types (ArrayBuffer, FormData, etc.) to checking for plain objects vs. everything else.